### PR TITLE
Move logging labels to the beginning of lines

### PR DESCRIPTION
**Pull request by happz**
This should improve readability. Sill plenty to improve, but it is better now.

### CODE REVIEW

File: tmt/log.py [hunk 1]
```diff
@@ -114,19 +114,20 @@ def indent(
...
+
+    # Multiline content indented deeper
+    lines = value.splitlines()
 
-        message = f'{prefix}{key}: {value}'
+    if len(lines) <= 1:
+        return f'{prefix}{indent}{key}: {value}'
 
-    return indent + message
+    return f'{prefix}{indent}{key}: {lines.pop(0)}\n' \
```
Comment (thrix) [inline tmt/log.py:129]: Can we add here a comment, why this is the way it is, the old form was in this more readable to me

**Review by thrix — CHANGES_REQUESTED** [review]

File: tmt/log.py (hunk 1 above)
Comment (happz) [reply tmt/log.py:129]: I'm not sure I follow, what do you want me to address?

Comment (thrix) [reply tmt/log.py:129]: nah, it is written actually in the function help.

**Review by thrix — APPROVED** [review]
LGTM

File: tmt/log.py (hunk 1 above)
Comment (janhavlin) [reply tmt/log.py:187]: To me the `pop(0)` call is obvious - we're treating the first line differently than the rest of the list. Looking at the string, I could just see a better description of what `prefix` variable is, the rest seems pretty well explained to me.

Comment (happz) [reply tmt/log.py:187]: It's actually a regression - the original code did not treat the first line differently. I misunderstood its behavior, fixing it right now.

Comment (happz) [reply tmt/log.py]: We want this:

```
some-key: single value
another-key:
    multiline
    string
    is here
```

and my code was printing this:

```
some-key: single value
another-key: multiline
    string
    is here
```

Fixed now.

Comment (thrix) [reply tmt/log.py]: so we misunderstood original code

Comment (happz) [reply tmt/log.py]: Added a unit test in 6807c8d4

File: tests/unit/test_logging.py [hunk 2]
```diff
@@ -221,6 +219,43 @@ def test_labels(caplog: _pytest.logging.LogCaptureFixture, root_logger: Logger)
 
     _exercise_logger(caplog, root_logger, labels=['foo', 'bar'])
 
+    _exercise_logger(caplog, root_logger, labels=['foo', 'bar'])
```
Comment (psss) [inline tests/unit/test_logging.py:222]: Why is this line duplicated?

**Review by psss — COMMENTED** [review]
Looks good, just one question. Also proposing a minor style adjustment in 2defd041.

File: tests/unit/test_logging.py (hunk 2 above)
Comment (happz) [reply tests/unit/test_logging.py]: No idea, removed in bb4a6670

**Review by psss — APPROVED** [review]

**Pull request merged**